Casks/l/linqpad@beta.rb

```diff
@@ -0,0 +1,34 @@
+cask "linqpad@beta" do
```

Comment:
```suggestion
cask "linqpad" do
```

As this is the "most stable" version available, we don't need to append the `@beta` suffix.
Please also update the filename to match.